### PR TITLE
Update Members.md

### PR DESCRIPTION
**Pull request by ritiksoni00**

### CODE REVIEW

**Comment by ritiksoni00** [comment]
@seema1711 sorry i'm late!

File: Members.md [hunk 1]
```diff
@@ -199,3 +199,12 @@ __Tech Stack__:  C/C++, Java, Kotlin, Firebase
...
 
+
+
+
+- __Name__: Ritik Soni   
+__Country__: India  
+__About__: to be 21 in upcoming feb, Junior Year of bechelors. Danjgo maybe... learning stuff about CPP and Python frameworks
+__Social Handles__: [Twitter](https://twitter.com/Arre_ritik) | [GitHub](https://github.com/ritiksoni00) | [LinkedIn](https://www.linkedin.com/in/ritik-soni/)   
+
+
```
Comment (ahammadshawki8) [inline Members.md:210]:
```suggestion
- __Name__: Ritik Soni   
__Country__: India  
__About__: to be 21 in upcoming Feb, Junior Year of bachelors. Danjgo maybe... learning stuff about CPP and Python frameworks.   
__Social Handles__: [Twitter](https://twitter.com/Arre_ritik) | [GitHub](https://github.com/ritiksoni00) | [LinkedIn](https://www.linkedin.com/in/ritik-soni/)   
```

**Review by ahammadshawki8 — CHANGES_REQUESTED** [review]
Great, Please solve the problems before merging it to the main branch.

File: Members.md (hunk 1 above)
Comment (ritiksoni00) [reply Members.md]: done!

**Comment by ahammadshawki8** [comment]
Now it is ready for merging ✌

File: Members.md [hunk 2]
```diff
@@ -199,3 +199,7 @@ __Tech Stack__:  C/C++, Java, Kotlin, Firebase
 __Social Handles__:  [LinkedIn](https://www.linkedin.com/in/rahulsain/)  | [GitHub](https://github.com/rahulsain) | [Twitter](https://twitter.com/rahul_sain01) | [Instagram](instagram.com/_rahulsain/)  
 __Website__:  [Podcast](https://anchor.fm/rahulsain) | [Listen to my Podcast on Spotify](https://open.spotify.com/show/1CJoDKXx9tmk5BxI0pvgGX)  
 
+- __Name__: Ritik Soni   
+__Country__: India
```
Comment (plazzy99) [inline Members.md:203]: Please add *tech stack* and *website* if any

**Review by plazzy99 — CHANGES_REQUESTED** [review]

**Pull request merged**